README.md

```diff
@@ -1,130 +1,182 @@
+## Next Steps
+
+This SDK will help facilitate OAuth link generation, popup dialog creation, and Smartcar will handle user authentication and authorization. This SDK will not assist in exchanging authorization codes for an access token. Should you choose to self host the redirect this SDK will not assist with extracting the authorization code query parameter from the redirect URI.
+
+See the links below to various SDKs for help in exchanging the authorization code for an access token. If the backend language you want to use isn't listed below you can still use Smartcar you'll just have to make HTTP requests manually based on our [API specification](https://smartcar.com/docs#get-all-vehicles).
+
+- [Node](https://github.com/smartcar/node-sdk)
+- [Python](https://github.com/smartcar/python-sdk)
+- [Java](https://github.com/smartcar/java-sdk)
+
+### Usage
+
+### Import `sdk.js`
+
+Import `sdk.js` on your client. Call the `Smartcar` constructor then use one of its instance methods to setup the authorization flow.
 
-### Import `sdk.js`:
 ```html
-<script src="https://cdn.smartcar.com/javascript-sdk/sdk-1.0.0.js"></script>
+<script src="https://javascript-sdk.smartcar.com/sdk-2.0.0.js"></script>
 ```
 
-### Use `sdk.js`:
+### Use `sdk.js`
+
 Initialize:
+
 ```javascript
 const smartcar = new Smartcar({
   clientId: `your-client-id`,
   redirectUri: `your-redirect-uri`,
-  scope: ['read_vehicle_info', 'read_odometer'], // optional
-  onComplete: function() { // optional
-    window.location.reload();
+  // optional, defaults to all scopes
+  scope: ['read_vehicle_info', 'read_odometer'],
+  // if using smartcar hosting this is required and must take at least two
+  // arguments - error & code
+  // otherwise optional (will be called with the same arguments but is not
+  // required to handle them)
+  onComplete: function(error, code, state <optional>) {
+    // actions to take on completion of auth flow
+    // if using smartcar hosting this should send the code to your backend
+    // server to exchange for an access token
   },
-  development: false, // optional
+  development: false, // optional, defaults to false
+  useSmartcarHostedRedirect: true, // optional, defaults to false
 });
 ```
-The best placement for the above code is just before the closing `</body>` tag.
 
-To open the Smartcar OAuth dialog directly:
-```javascript
-// note: state and forcePrompt are optional
-smartcar.openDialog({state, forcePrompt});
-```
+<!-- TODO: why show state and forcePrompt in this? just seems to add unnecessary disclaimers -->
 
-To add a click handler to an HTML element that will open the Smartcar OAuth dialog:
-```javascript
-// note: state and forcePrompt are optional
-smartcar.addClickHandler({id, state, forcePrompt});
-```
+Once initialized there are three instance methods for setting up the OAuth flow:
```

Comment:
Do we want an example of where we would add any of the following instances. Tbh, I'd probably need that.

Reply:
Can you give an example of how we'd do that? Feels hard to tell them where to add something in their code without context.

Reply:
mm not sure, but as a random example -

> You can add `openDialog()` at the bottom of your `<script>` in `index.html`.

I'm not sure if that makes sense, but something along those lines will make it clear for readers at any level. If it doesn't make sense and there's no logical way of communicating it, feel free to ignore :)

Reply:
I guess my worry is that if they do that it might not be connected to their UX, like if they just stuck it at the bottom of their script it would just always run on page load which they probably shouldn't wouldn't (and arguably shouldn't) want